app/controllers/admin/dashboard_controller.rb

```diff
@@ -34,6 +35,26 @@ def prep_projects_grid
     @status_filter_options = STATUS_FILTERS.map { |f| [I18n.t("dashboard.status_options.#{f}"), f] }
   end
 
+  # Prepare grids for all users inside selected division
+  def prep_projects_grids
+    @people = @division.people.where(has_system_access: true).where.not(id: @person.id)
```

Comment:
Can you also please add a scope to Person called `with_system_access` and refactor any places that have the full `where` like you have. I think there are around three others.